examples/restore_dashboards.py

```diff
@@ -13,18 +13,33 @@
 #
 # Parse arguments
 #
-if len(sys.argv) != 3:
-    print 'usage: %s <sysdig-token> <file-name>' % sys.argv[0]
+if len(sys.argv) <= 2:
+    print 'usage: %s <sysdig-token> <file-name> <backend_url> <ssl_check>' % sys.argv[0]
     print 'You can find your token at https://app.sysdigcloud.com/#/settings/user'
+    print 'Optional: backend_url (https://app.sysdigcloud.com, https://app-staging.sysdigcloud.com)'
+    print 'Optional: ssl_check (True/False)'
     sys.exit(1)
 
 sdc_token = sys.argv[1]
 dashboard_state_file = sys.argv[2]
 
+try:
+    backend_url = sys.argv[3]
```

Comment:
If both 3rd and 4th parameter are optional, then `sys.argv[3]` could also be `True/False` so this code would :fire: